04.class/lib/input.js

```diff
@@ -0,0 +1,28 @@
+import readline from "readline";
+
+class Input {
+  constructor(option) {
+    this.option = null;
+    this.contens = [];
```

Comment:
`contents` の typo でしょうか？

```suggestion
    this.contents = [];
```